Framework/Kernel/src/Matrix.cpp

```diff
@@ -1613,7 +1613,7 @@ void fillFromStream(std::istream &is, Kernel::Matrix<T> &in,
 
 // Symbol definitions for common types
 template class MANTID_KERNEL_DLL Matrix<double>;
-template class MANTID_KERNEL_DLL Matrix<int>;
+template class Matrix<int>;
```

Comment:
Performing `nm -C -D libMantidKernel.so | grep "Mantid::Kernel::Matrix<int>::"` on the Kernel library produces an equivalent output to `nm -C -D libMantidKernel.so | grep "Mantid::Kernel::Matrix<double>::"`